src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -359,6 +372,17 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
+		}
+
+        internal void InitializeNamespacesToExcludeFromLibrary()
+        {
+            if (!NamespacesToExcludeFromLibrarySpecified)
+            {
+                NamespacesToExcludeFromLibrary.Add(
+                    "ProtoGeometry.dll:Autodesk.DesignScript.Geometry.TSpline"
```

Comment:
Can't we say if(! NamespacesToExcludeFromLibrary.Any()) instead of having a separate property?